packages/core/src/payment/strategies/klarnav2/klarnav2-payment-strategy.spec.ts

```diff
@@ -104,9 +101,6 @@ describe('KlarnaV2PaymentStrategy', () => {
         jest.spyOn(orderActionCreator, 'submitOrder')
             .mockReturnValue(submitOrderAction);
 
-        jest.spyOn(paymentMethodActionCreator, 'loadPaymentMethod')
-            .mockReturnValue(loadPaymentMethodAction);
-
```

Comment:
As I said in line: 162 you must create the mock for `PaymentMethodRequestSender.loadPaymentMethod`